default/buildings.txt

```diff
@@ -1575,6 +1622,8 @@ BuildingType
         Population low = [[MIN_RECOLONIZING_SIZE]]
         OwnedBy empire = Source.Owner
     ]
+    EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
```

Comment:
Currently the Colony Base and the Outpost Base are disabled, it's worth considering changing that with the new colony mechanics, but agree, I regularly queue multiple bases in a new system, definitely don't want to prevent that.